esp/scm/ws_logaccess.ecm

```diff
@@ -25,7 +25,9 @@ ESPenum LogAccessType : int
     BySourceInstance(5, "BySourceInstance"),
     BySourceNode(6, "BySourceNode"),
     ByFieldName(7, "ByFieldName"),
-    ByPod(8, "ByPod")
+    ByPod(8, "ByPod"),
```

Comment:
@asselitx this PR adds a couple of entries in a number of ECM enumerations which I want to annotate w/ the appropriate interface version, but I found no mechanism to do so. Is there a way to declare the new entries to require at least client version 1.06+ ? Thanks.